covid_model/model_fit.py

```diff
@@ -80,18 +87,13 @@ def run(self, engine, method='curve_fit', window_size=14, look_back=None,
             this_end_t = tslices[-trim_off_end] if trim_off_end > 0 else end_t
             this_end_date = self.base_specs.start_date + dt.timedelta(days=this_end_t)
 
-            model = CovidModel(base_model=base_model, end_date=this_end_date)
+            t01 = perf_counter()
+            model = CovidModel(base_model=base_model, end_date=this_end_date, deepcopy_params=False)
             model.apply_tc(tc[:len(tc)-trim_off_end], tslices=tslices[:len(tslices)-trim_off_end])
+            t02 = perf_counter()
+            print(f'Model copied in {t02-t01} seconds.')
```

Comment:
I added this because of the horrible problems I was happening with the copying being slow. It's still taking like 3 seconds. I feel like it's worth leaving this here for now, to keep an eye to make sure it doesn't creep up again.